instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -212,6 +219,9 @@ def instrument_app(
         tracer_provider=None,
         meter_provider=None,
         excluded_urls=None,
+        http_capture_headers_server_request: list[str] | None = None,
```

Comment:
Any reason why we are not adding this feature to the `instrument()` method? It seems consistent with the other configuration features like `"server_request_hook"` and `"client_request_hook"`

Reply:
good point, I've added the same to `instrument()` and tests.